test-clients/src/main/java/com/hedera/services/yahcli/Yahcli.java

```diff
@@ -116,6 +115,10 @@ public String getPayer() {
         return payer;
     }
 
+    public boolean getSchedule() {
+        return schedule;
+    }
```

Comment:
should be `isScheduled` or `isScheduleEnabled` ?

Reply:
isScheduled I changed it